proposals/incubation/credebl.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: default
+title: CREDEBL
```

Comment:
One of the things that will need to be determined is trademark. If you wish to continue using CREDEBL as a trademark for a product, you will need to come up with a new name for contributing to LF Decentralized Trust.

Reply:
AYANWORKS does not have trademark on CREDEBL. We wish to use CREDEBL as a project name under LFDT, subject to the legal trademark approval. We plan to use a new name for the product.

Reply:
Also responded to David over email.